apps/damap-frontend/src/index.html

```diff
@@ -6,7 +6,7 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
-  <body>
+  <body style="zoom: 90% !important; background-color: #f6f6f6 !important">
```

Comment:
You should use a variable for the color, i believe there alreay is one with the exact same color. Also, is there a reason the zoom is needed? If the effect should stay, we should replace the zoom property, read more [here](https://css-tricks.com/almanac/properties/z/zoom/).

Reply:
No, i just asked out of curiosity 😄  - although we should still replace zoom with a more suitable alternative.